packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -160,6 +162,24 @@ export class DocumentDeltaConnection
 					this.emit(event, ...args);
 				});
 			}
+
+			// log latency every minute
+			this.trackLatencyTimer = setInterval(() => {
+				const start = Date.now();
+
+				// emit pong event every minute. If latency is longer than 1 min, log separately
+				this.socket.emit("pong", () => {
+					const latency = Date.now() - start;
+					if (latency > 1000 * 60) {
+						this.mc.logger.sendPerformanceEvent({
```

Comment:
I'd suggest using error, not generic event here.
Also it's better to use `duration`, not `latency` as a name of a field, to be more uniform with other time-measurement events.